src/hooks/stations.ts

```diff
@@ -37,10 +38,25 @@ export const useBooking = () => {
         }),
     [BUCHUNGEN],
   );
+  const refreshBooking = useCallback(
+    () =>
+      getCurrentBooking()
+        .then(setBooking)
+        .then(() => {
+          const oldBooking = booking;
+          return cancelCurrentBooking()
+            .then(oldBooking && (() => bookBike(oldBooking.stationId)))
```

Comment:
I think this should book a bike even in the case we don't have a previous booking. When the user presses the button and the booking has gone this is because the booking timed out & when the user presses refresh he wants a booking.

Reply:
Then the data has to be obtained from somewhere else, the selected station would be a good option.

Reply:
But we should already have a booking stored inside the hook at this point. That contains the station ID.